packages/rum-recorder/src/domain/rrweb/record.ts

```diff
@@ -260,11 +261,9 @@ export function record(options: RecordOptions = {}): RecordAPI {
       )
     )
   }
-  if (document.readyState === 'interactive' || document.readyState === 'complete') {
-    init()
-  } else {
-    handlers.push(on('load', init, window))
-  }
+
+  runOnReadyState('complete', init)
+
```

Comment:
Just to be sure I understand the confusion here, it should have used `on('DOMContentLoaded',...)` instead of `on('load',...)` to be consistent, right?

Reply:
Yes that's it :)